erpnext/selling/doctype/sales_order/sales_order.py

```diff
@@ -71,25 +70,6 @@ def validate_po(self):
 				"allow_against_multiple_purchase_orders")):
 				frappe.msgprint(_("Warning: Sales Order {0} already exists against Customer's Purchase Order {1}").format(so[0][0], self.po_no))
 
-	def validate_for_items(self):
-		check_list = []
-		for d in self.get('items'):
-			check_list.append(cstr(d.item_code))
-
-			# used for production plan
-			d.transaction_date = self.transaction_date
-
-			tot_avail_qty = frappe.db.sql("select projected_qty from `tabBin` \
-				where item_code = %s and warehouse = %s", (d.item_code, d.warehouse))
-			d.projected_qty = tot_avail_qty and flt(tot_avail_qty[0][0]) or 0
```

Comment:
@deepeshgarg007 Set `transaction_date` and `projected_qty` via a separate function, as you have removed that.